js/banner.js

```diff
@@ -1,9 +1,8 @@
 window._idl = {};
-    _idl.variant = "banner";
-    (function() {
-        var idl = document.createElement('script');
-        idl.type = 'text/javascript';
-        idl.async = true;
-        idl.src = ('https:' == document.location.protocol ? 'https://' : 'http://') + 'members.internetdefenseleague.org/include/?url=' + (_idl.url || '') + '&campaign=' + (_idl.campaign || '') + '&variant=' + (_idl.variant || 'banner');
-        document.getElementsByTagName('body')[0].appendChild(idl);
-    })();
\ No newline at end of file
+	_idl.variant = "banner";
+	(function() {
+	    var idl = document.createElement('script');
+	    idl.async = true;
+	    idl.src = 'https://members.internetdefenseleague.org/include/?url=' + (_idl.url || '') + '&campaign=' + (_idl.campaign || '') + '&variant=' + (_idl.variant || 'modal');
```

Comment:
As I'm looking at these now (however many years later) I think this bit is superfluous: `(_idl.variant || 'modal')` because `_idl.variant` gets defined on line 2. There's never going to be a case where it's undefined, so we might as well pull out the `||` on line 6 when we're building the URL.